src/app/wallets/get-account-transactions-for-contact.ts

```diff
@@ -0,0 +1,30 @@
+import { LedgerService } from "@services/ledger"
+import { toLiabilitiesAccountId, LedgerError } from "@domain/ledger"
+import { WalletTransactionHistory } from "@domain/wallets"
+import { PartialResult } from "@app/partial-result"
```

Comment:
I thought we didnt want to use this pattern because of all the problems detected at the beginning of the refactor. Why do we need this?

Reply:
We are using for the main transactions list, and this PR follows along.